packages/ui-components/src/components/modal/modal.base.scss

```diff
@@ -18,7 +18,7 @@
 	--modal-min-height: 235px;
 	--modal-width: fit-content;
 	--modal-min-width: 400px;
-	--modal-topbar-height: 60px;
+	--modal-topbar-height: 51px;
```

Comment:
is it not 50px? 49px + 1px border bottom?

Reply:
Yes 😄